docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,119 @@
 
 ## Known Issues for PowerShell on Non-Windows Platforms
 
-The first Alpha release of PowerShell on Linux and macOS is mostly functional but
-does have some significant limitations and usability issues.
+Alpha releases of PowerShell on Linux and macOS are mostly functional but do have some significant limitations and usability issues.
```

Comment:
Link to query would be sufficient.  Just seems odd to reference Alpha as we are getting close to leaving Alpha state.